### DISCUSSION TITLE
Universal G-code Sender opens code but is not visible in the viewer and does not allow sending

### DISCUSSION THREAD

**Discussion by TrainerLion** [winder/Universal-G-Code-Sender#2451] [Help] [answered]
I bought a CNC 3018 pro, I'm new to this, I have a problem with Universal G-code Sender, my problem is that when I open the code the visualization is only generated for a few seconds and disappears, to create the code I used vetric aspire, and it doesn't work for me, I would appreciate it very much if someone can help me, thank you

**Comment by breiler** [comment]
Can I have your log file, open the menu `Tools / Open Log directory` and attach the `messages.log`.

It might be helpful to get one of your gcode file.

**Reply by TrainerLion** [reply]
[messages.log](https://github.com/winder/Universal-G-Code-Sender/files/14171574/messages.log)


[GCode ronaldo.zip](https://github.com/winder/Universal-G-Code-Sender/files/14171579/GCode.ronaldo.zip)

Here are the files friend thanks.

**Reply by breiler** [reply, marked as answer]
You are getting a bunch of these errors:
`Caused: java.nio.file.NoSuchFileException: C:\Users\wlope\AppData\Local\Temp\Rar$EXa29216.37520\ugsplatform-win\ide\modules\org-netbeans-modules-editor-fold-nbui.jar`

Did you perhaps try to start the program from within WinRAR without properly extracting the zip?

Remove the directory `ugsplatform-win` from your desktop and unzip it again. 

If it still doesn't work, close UGS and delete the folders `C:\Users\wlope\AppData\Roaming\ugsplatform` and `C:\Users\wlope\AppData\Local\ugsplatform\Cache\var\cache` and start UGS again.

**Reply by TrainerLion** [reply]
I tried all your tips and what worked for me was to delete the folders and restart, thanks for your help.
